QueueInteropTransport.php

```diff
@@ -168,6 +170,16 @@ public function send(Envelope $envelope): Envelope
             $producer->setTimeToLive($this->options['timeToLive']);
         }
 
+        $partitionStamp = $envelope->last(PartitionStamp::class);
+        if (method_exists($interopMessage, 'setPartition') && !is_null($partitionStamp)) {
+            $interopMessage->setPartition($partitionStamp->getPartition());
```

Comment:
What class do you expect here? 
Ie, it is not `Interop\Queue\Message`, right?

Reply:
@Nyholm do you prefer that?

```php
$partitionStamp = $envelope->last(PartitionStamp::class);
if ($interopMessage instanceof RdKafkaMessage) && !is_null($partitionStamp)) {
    $interopMessage->setPartition($partitionStamp->getPartition());
}
```

Reply:
ping @Nyholm @sroze